### PR TITLE
Docs 0.0.2

### PR DESCRIPTION
**Pull request by ashcrow**
I can't say that I found everything that needs updating but I believe I got most of it.

### CODE REVIEW

File: doc/services_devel.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        pass
+
+
+Define Exposed methods
+``````````````````````
+
+``CommissaireService`` uses the ``on_{{ method }}`` convention for exposing methods
+to remote callers. If you wanted to expose a method as ``ping`` you would
+define a method on your service called ``on_ping``. ``on_{{ method }}``'s
+expect to take 1 or more arguments where the 1 required argument is ``message``
```
Comment (mbarnes) [inline doc/services_devel.rst]: Typo: 1 -> 1st or first

File: doc/services_devel.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+which is the message itself in the case the method needs extra information.
+
+To return results back to the caller via the message bus simply use the ``return``
+statement as if it was a normal method. If there is an error ``raise`` the
```
Comment (mbarnes) [inline doc/services_devel.rst]: nit: Add a comma after "error".

**Review by mbarnes — CHANGES_REQUESTED** [review]
Changes look good, aside from a couple nits in `services_devel.rst`.

If you don't mind, could you also substitute `cluster_manager` with `container_manager` in `doc/endpoints.rst`.  Noticed that error just today.

**Comment by ashcrow** [comment]
Will update shortly.

**Comment by ashcrow** [comment]
:arrow_up:

**Review by mbarnes — APPROVED** [review]

**Comment by mbarnes** [comment]
Squashed and merged manually.

**Pull request closed without merging**